### PR TITLE
Added rtpr, memlock & msgq rlimits

### PR DESCRIPTION
**Pull request by eli-zr**
Expanding rlimit interface with realtime priority, locked memory and msg queue size

### CODE REVIEW

File: cmdline.cc [hunk 1]
```diff
@@ -426,6 +429,9 @@ std::unique_ptr<nsjconf_t> parseArgs(int argc, char* argv[]) {
 	nsjconf->rl_nofile = 32ULL;
 	nsjconf->rl_nproc = parseRLimit(RLIMIT_NPROC, "soft", 1);
 	nsjconf->rl_stack = parseRLimit(RLIMIT_STACK, "soft", 1);
+	nsjconf->rl_mlock = parseRLimit(RLIMIT_MEMLOCK, "soft", 1);
+	nsjconf->rl_rtpr = 0;
```
Comment (happyCoder92) [inline cmdline.cc:433]: I think defaulting to "soft" might make more sense for all the newly added limits

File: cmdline.cc [hunk 2]
```diff
@@ -109,6 +109,9 @@ struct custom_option custom_opts[] = {
     { { "rlimit_nofile", required_argument, NULL, 0x0205 }, "RLIMIT_NOFILE, 'max' or 'hard' for the current hard limit, 'def' or 'soft' for the current soft limit, 'inf' for RLIM64_INFINITY (default: 32)" },
     { { "rlimit_nproc", required_argument, NULL, 0x0206 }, "RLIMIT_NPROC, 'max' or 'hard' for the current hard limit, 'def' or 'soft' for the current soft limit, 'inf' for RLIM64_INFINITY (default: 'soft')" },
     { { "rlimit_stack", required_argument, NULL, 0x0207 }, "RLIMIT_STACK in MB, 'max' or 'hard' for the current hard limit, 'def' or 'soft' for the current soft limit, 'inf' for RLIM64_INFINITY (default: 'soft')" },
+    { { "rlimit_mlock", required_argument, NULL, 0x0209 }, "RLIMIT_MEMLOCK in KB, 'max' or 'hard' for the current hard limit, 'def' or 'soft' for the current soft limit, 'inf' for RLIM64_INFINITY (default: 'soft')" },
```
Comment (happyCoder92) [inline cmdline.cc:112]: keep cmdline arg names consistent with the RLIMIT_* constant names.

Comment (eli-zr) [reply cmdline.cc]: Pushed a fix. Thanks.

File: cmdline.cc (hunk 1 above)
Comment (eli-zr) [reply cmdline.cc]: Agree.

**Review by happyCoder92 — APPROVED** [review]

**Pull request merged**